contracts/governance/GraphGovernanceStorage.sol

```diff
@@ -0,0 +1,20 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.7.3;
+
+import "./Governed.sol";
+import "./IGraphGovernance.sol";
+
+contract GraphGovernanceV1Storage is Governed {
```

Comment:
A design decision we are making is not including this Contract in the Controller. Which i think is the right choice.

But i would like to briefly discuss what we plan to do for upgrading the contract. This seems like the plan:
- It follows normal proxy pattern
- the owner is the graph council multisig
- we upgrade the code when needed.

The other option would be, starting a new contract from scratch, and stopping using the V1. I would prefer to use this naturally. BUT - because all of our other contracts use the proxy pattern. I am cool with keeping this as proxy. 

Sounds good? @abarmat  ?
Do you agree

Reply:
The contract will be deployed like a pair (GraphProxy -> GraphGovernance) so we can change the implementation in the future. What we can do is to set the GraphProxy admin to our current GraphProxyAdmin contract so we can manage upgrades from the same place like the rest.

So the final setup would be:

**GraphProxyAdmin -> (owns) -> GraphProxy -> (uses) -> GraphGovernance**